core/src/main/java/com/linecorp/armeria/common/util/TimeoutController.java

```diff
@@ -45,17 +49,24 @@
      *
      * @param newTimeoutMillis the new timeout value in milliseconds. {@code 0} if disabled.
      * @return {@code true} if the current timeout is reset by the specified {@code newTimeoutMillis}.
+     *         {@code false} if the current timeout could not be reset
+     *         because of some constraints in the class that implements this interface.
```

Comment:
What are the 'some constraints'?

Reply:
The `resetTimeout` is failed when `TimeoutTask` can not schedule.
I've tried to explain the exact cause... but `TimeoutTask` is only used in `DefaultTimeoutController`.
In this case, do I need to remove this sentence and simplify with: 
```
{@code false} if the current timeout failed to reset
```
or remove the ambiguous `{@code false} ...` sentence?